waiter/project.clj

```diff
@@ -30,7 +30,8 @@
                    :dev :dev
                    :perf (every-pred :perf (complement :explicit))}
 
-  :dependencies [[bidi "2.1.5"
+  :dependencies [[reaver "0.1.2"]
```

Comment:
Please add this explicitly as a test dependency by adding `:scope "test"`.

Nitpick: lexicographic order

Reply:
done